src/components/common/Filter/FilterComponent.jsx

```diff
@@ -50,10 +37,11 @@ class FilterComponent extends React.Component {
           />
           <Accordion.Content active={activeIndex === index}>
             <Form className="filter-form">
-              {
-                option.content.map((opt) => {
+              <Form.Group grouped>
```

Comment:
The addition of this has messed with the styling that we had before. The filters look very squeezed now. kindly revert to the original